nss-crystal-cobras/BangazonSite
# Ac product detail

Closes Ticket #2 

# Description

Created ProductDetails.cshtml view within Views/Home, and there is an identical Details.cshtml view within Views/Products. Functionally it will look the same as the previous ticket, but the routing is dynamic and set up for better future functionality. Also added user authentication code to the ProductsController.

## Type of change

Please delete options that are not relevant.


# Testing Instructions

git fetch --all 
git checkout ac-product-detail
run Bangazon
check that each product listed on the home page links to the product's respective details view


# Checklist:
